patsylearn/tests/test_patsy_model.py

```diff
@@ -124,3 +139,19 @@ def check_centering(X):
     est.estimator_.check_X = check_centering
     # make sure that mean of training, not test data was removed
     est.predict(data_test)
+
+
+def test_transform_to_dataframe():
+    data_train = pd.DataFrame(patsy.demo_data("x1", "x2", "y"))
+    data_train['x1'][:] = 1
+    # mean of x1 is 1
+    data_test = pd.DataFrame(patsy.demo_data("x1", "x2", "y"))
+    data_test['x1'][:] = 0
+
+    # center x1
+    est = PatsyTransformer("center(x1) + x2", return_type='dataframe')
+    est.fit(data_train)
```

Comment:
is there a reason why you need a separate ``data_trans``? And why do you set any features to a fixed value?